src/olink/remote/node.py

```diff
@@ -43,7 +43,9 @@ def handle_invoke(self, id: int, name: str, args: list[Any]) -> None:
         source = self.get_source(name)
         if source:
             value = source.olink_invoke(name, args)
-            self.emit_write(Protocol.invoke_reply_message(id, name, value))
+            # only send reply for not empty values
+            if value:
```

Comment:
value could be int 0. I guess we would like to return an int 0 value or empty string. I guess you need to know if the `value == None`. Seems to be more safe